### ISSUE TITLE
Accessibility of project properties window

### ISSUE THREAD

**Issue by FayCross** [thexerteproject/xerteonlinetoolkits#1122]
Issues with accessibility of project properties window:

- [ ] Can't change tab using keyboard (none of the tabs can be tabbed to)
- [ ] All the checkboxes are images so can't be tabbed to or changed via keyboard
- [ ] White text on orange background buttons don't pass colour contrast checks

**Comment by FayCross** [comment]
I've done some work on this so should be ready for 3.13
